### PR TITLE
Spelling and grammar fixes

### PR DESCRIPTION
**Pull request by articlecat**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -314,7 +314,7 @@ Remember: Without strong encryption, you will be spied on systematically by lots
 **Check out [this site](https://www.securemessagingapps.com/) for comparisons*.
 
 <img width="16" src="misc/forbidden.png"> </img> **Avoid**
-- Whatsapp
+- WhatAapp
```
Comment (pluja) [inline README.md:317]: Thanks for the fixes, could you correct this typo here before I merge this?

File: README.md [hunk 2]
```diff
@@ -558,7 +558,7 @@ If you need an app for **menstrual cycle tracking** please don't use any apps li
 - [ProExpense](https://github.com/arduia/ProExpense/) - A simple free finance note to safely record daily expenses.
 - [IHateMoney](https://ihatemoney.org/) - Manage your shared expenses, easily.
 - [Nextcloud Cospend](https://apps.nextcloud.com/apps/cospend) - A group/shared budget manager inspired by the great IHateMoney.
-   - [MoneyBuster](https://gitlab.com/eneiluj/moneybuster/) - Android client for Nextcloud Cospend and [IHateMoney](https://ihatemoney.org/) servers.
+   - [MoneyBuster](https://gitlab.com/eneiluj/moneybuster/) - Android client for Nextcloud, Cospend and [IHateMoney](https://ihatemoney.org/) servers.
```
Comment (pluja) [inline README.md:561]: I think it is Nextcloud Cospend:
* https://github.com/eneiluj/cospend-nc
* https://apps.nextcloud.com/apps/cospend

**Review by pluja — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (articlecat) [reply README.md]: Done, thank you!

File: README.md (hunk 2 above)
Comment (articlecat) [reply README.md]: Fixed, thanks!

**Review by pluja — APPROVED** [review]

**Pull request merged**